src/Data/LLVM/BitCode/IR/Constants.hs

```diff
@@ -461,6 +431,20 @@ parseConstantEntry t (getTy,cs) (fromEntry -> Just r) =
     ty <- getTy
     return (getTy, Typed ty ValPoison : cs)
 
+  27 -> label "CST_CODE_DSO_LOCAL_EQUIVALENT" $ do
+    notImplemented
```

Comment:
(I know this PR is still a draft, but): Should we make an issue for this, and the other `notImplemented` below?

Reply:
Yes, certainly. I'll do this in due course.

Reply:
Great, thank you!

Reply:
I've opened #208 and #209 for these.